sdk/messaging/azeventhubs/checkpoint_store.go

```diff
@@ -31,9 +33,9 @@ type Ownership struct {
 	FullyQualifiedNamespace string
 	PartitionID             string
 
-	OwnerID          string    // the owner ID of the Processor
-	LastModifiedTime time.Time // used when calculating if ownership has expired
-	ETag             string    // the ETag, used when attempting to claim or update ownership of a partition.
+	OwnerID          string      // the owner ID of the Processor
+	LastModifiedTime time.Time   // used when calculating if ownership has expired
+	ETag             azcore.ETag // the ETag, used when attempting to claim or update ownership of a partition.
```

Comment:
Does this need to be `*azcore.ETag`?  I know we use that in other places as I believe that an empty-string ETag is semantically different from no ETag.

Reply:
Ah, interesting. I'm good with being explicit, I'll switch it over.

Reply:
You might want to double-check how this is used in other places, but I'm pretty sure you'll want this change.

Reply:
Swapped over, seemed clean enough. I don't intend to interpret this value, so I think it's best to just completely stay out of that game on the client.